tests/test_cli.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import os
+import signal
+import threading
+import time
+
+from prometheus_pgbouncer_exporter import cli
+from unittest import mock
+
+
+CURR_DIR=os.path.dirname(os.path.realpath(__file__))
+
+
+class TestCLI(unittest.TestCase):
+
+    def testReadConfigFile(self):
```

Comment:
This test is verbose and that's something we should avoid. Please mock the logger object or adjust the log level to avoid message